README.md

```diff
@@ -12,7 +12,7 @@ Quickly translate text right in your code 🚀
 ### Translate selected text
 
 1. Select some text to translate
-1. Press `ALT+SHIFT+T`
+1. Press `CTRL+ALT+T`
```

Comment:
Why don't we keep the current shortcut?

Reply:
mostly because I was debugging why it didn't work. And I never changed it back after testing some alternatives

Reply:
Can you please investigate why the first shortcut wasnt working? Maybe another extension or something else?

Reply:
So there was 2 reasons why it didn't work. The declaration wasn't in the "contributes" part of the package.json file. It also said  "alt+shit+t" instead of "alt+shiFt+t". A comical typo that brought a smile to my face :)

Reply:
Ahah 😂 nice typo. Ok for CTRL+ALT+T then.